packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -131,13 +121,25 @@ export default class QuanticCaseClassification extends LightningElement {
   _value = '';
   /** @type {boolean} */
   _isSuggestionsVisible = true;
+  /** @type {number} */
+  _maxSuggestions = 3;
+  /** @type {boolean} */
+  _renderingError = false;
 
   connectedCallback() {
     registerComponentForInit(this, this.engineId);
```

Comment:
Why bother registering the component if it won't be initialized due to the missing attribute?